src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMA = ",";
+    private int supplySum = 0;
+    private int buySum = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        readFile(fromFileName);
+        String report = createReport();
+        writeFile(toFileName, report);
+    }
+
+    private String createReport() {
+        StringBuilder builder = new StringBuilder();
+        return builder.append(SUPPLY)
+                .append(COMA)
+                .append(supplySum)
+                .append(System.lineSeparator())
+                .append(BUY)
+                .append(COMA)
+                .append(buySum)
+                .append(System.lineSeparator())
+                .append(RESULT)
+                .append(COMA)
+                .append(supplySum - buySum)
+                .append(System.lineSeparator()).toString();
+    }
+
+    private void readFile(String fromFileName) {
```

Comment:
This method doing 2 things: 
1. Reading data from file
2. Calculating supplySum  and buySum 
It will be better lo separate this responsibilities or name this method readFileAndCalculateStatictics, I prefere first solution